.pmz.yaml.example

```diff
@@ -1,5 +1,6 @@
 editor: your editor path. Example: /usr/bin/vim
 ztldir: your zettel directory - it must exist! Example: /home/<user>/zettel
+notetmpl: if you have a note template you would like to use. Example: /home/<user>/zettel_tmpl.txt
```

Comment:
I added `notetmpl: /path/to/template` and ran `pmz new` -- it did not pre-fill the template.  I ran `pmz --notetemplate /path/to/template` and it worked just fine!

Maybe because the source code is not reading the `notetmpl` variable in the yaml file?  https://github.com/gsilvapt/pmz/blob/999e3368f238e795c6111bcabc911312e99532cf/cmd/new.go#L45 is looking for "notetemplate", is that why?

Instead of `notetmpl` in `.pmz.yaml`, I used `notetemplate` and it worked fine, in which case just this example file needs to be changed for consisten variables?

Reply:
Yes, you are right. I misspelled in the configuration sample and that's why it worked with the flag.

I should start reading how to write unit tests for these type of applications :sweat_smile: 

Thanks for the review. Will push a fix ASAP.